Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -634,6 +634,15 @@ private void updateNavigationDrawerItems() {
 
         if (mTagsAdapter.getCountCustom() > 0) {
             mNavigationMenu.add(GROUP_SECONDARY, TAGS_ID, Menu.NONE, getString(R.string.tags)).setActionView(R.layout.drawer_action_edit).setEnabled(false);
```

Comment:
Warning (⚠️): Consider removing this line as it is being replaced by the below one. If this line is not removed then the drawer will show two `Tags EDIT` rows, with the first one now working as expected.

Reply:
Nice catch!  That looks like a copy/paste error.  I removed the redundant line in f7144fb3.

Reply:
Coolio, many thanks! 🌟